src/test/java/com/documents/LicenseRegistrationTests.java

```diff
@@ -217,6 +226,98 @@ public void functionality_licenseRequest_findAll_should_return_list_of_licenseRe
 
     }
 
+    @Test
+    public void behavioural_create_pdf() throws Exception {
+
+        List<String> infoList = new ArrayList<>();
+        infoList.add("Ciofu Vlad");
+        infoList.add("118932 Salveaza-l acum");
+        infoList.add("2018");
+        infoList.add("3 ");
+        infoList.add("");
+        infoList.add("10 Ianuarie");
+        infoList.add("$$$");
+        infoList.add("");
+        infoList.add("");
+        infoList.add(" Program anti anti plagiere");
+        infoList.add(" Cine ma primeste");
+        infoList.add("10-06-2018");
+
+        licenseRegistrationFormServiceImpl.createPdf(infoList, "src\\main\\resources\\test.pdf");
+
+        assertEquals(true, new File("src\\main\\resources\\test.pdf").exists());
+
+        //licenseRegistrationFormServiceImpl.createPdf(infoList, "D:/test.pdf");
+
+        //assertEquals(true, new File("D:/test.pdf").exists());
+    }
+
+    @Test
+    public void functionality_create_pdf() throws Exception {
+
+        List<String> infoList = new ArrayList<>();
+        infoList.add("Ciofu Vlad");
+        infoList.add("118932 Salveaza-l acum");
+        infoList.add("2018");
+        infoList.add("3 ");
+        infoList.add("");
+        infoList.add("10 Ianuarie");
+        infoList.add("$$$");
+        infoList.add("");
+        infoList.add("");
+        infoList.add(" Program anti anti plagiere");
+        infoList.add(" Cine ma primeste");
+        infoList.add("10-06-2018");
+
+        licenseRegistrationFormServiceImpl.createPdf(infoList, "src\\main\\resources\\test.pdf");
+
+        assertEquals(true, new File("src\\main\\resources\\test.pdf").exists());
+
+        PdfReader pdfReader = new PdfReader("src\\main\\resources\\test.pdf");
+
+        String content = PdfTextExtractor.getTextFromPage( pdfReader, 1 );
+
+        pdfReader.close();
+
+        BufferedReader br = new BufferedReader(new FileReader("src\\main\\resources\\documentsAdditionalText.json"));
+        String jsonString;
+
+        jsonString = PdfUtility.initiliazeJson(br);
+
+        JSONObject obj = new JSONObject(jsonString);
```

Comment:
this too :)... don't be cheap on letters :)

And change the song, please :)....it makes ears bleed :(